---
layout: home
title: DREAM Project Site
---

## About Me

I am a Master’s in Computer Science candidate at Tufts University in Medford, MA. Prior to my arrival at Tufts, I spent three years working at a literary agency in Brooklyn, NY. I earned a B.S. in Biology from The University of Alabama in 2016, where I wrote my honors senior thesis on the use of liquid chromatography-mass spectrometry (LC-MS) to detect endocrine-disrupting compounds in mangrove rivulus (Kryptolebias marmoratus) tissues.
 
While at Tufts, I’ve been active in computational biology research: I’ve contributed to the development of a disease gene prioritization algorithm and am currently working on a project analyzing DNA repair. I am expanding on the DNA repair analysis for the DREAM program this summer.
 
In my spare time, I bake, hike, and draw. I plan to graduate with my M.S. in Computer Science in May 2023.


## About My Advisor

[Lenore Cowen](http://www.cs.tufts.edu/~cowen/) is a professor of Computer Science at Tufts University. She holds a B.A. in Mathematics from Yale and a Ph.D. in Mathematics from MIT. Her primary research areas are in graphs, networks, and algorithms and their applications in computational molecular biology. 

## About My Project

DNA double-stranded breaks (DSBs) are very common and caused by a multitude of events, from chemotherapy treatments to errors in DNA replication. Improper repair can be detrimental to the cell and, in some cases, to human health (1). Repair of these breaks occurs through a variety of mechanisms that depend on the phase of the cell cycle, presence of helper molecules, and characteristics of the sequence undergoing repair. In some of these mechanisms, a single-stranded template is available, and repair results are highly accurate. In others, a diverse set of often poorly-understood and error-prone processes can occur (2). 

The goal of my research this summer will be to characterize DSB repair both broadly and in the context of a specific error-prone pathway known as synthesis-dependent microhomology-mediated end-joining (SD-MMEJ). Given a dataset of ~1,100 partly-randomized DNA constructs subject to Cas9-induced double-stranded breaks and subsequently repaired and sequenced, I will explore whether we can use sequence characteristics to characterize patterns in erroneous repair. A broad overview of the project splits this problem into two parts:

1.	Cluster constructs into bins based on sequence characteristics
2.	Examine whether constructs within the same bin share similar “repair events” or mutations that occur during repair

I will explore and analyze the results of various approaches to these two aims. This project is being developed in collaboration with the [McVey Lab](https://ase.tufts.edu/biology/labs/mcvey/), which studies DNA damage and repair using _Drosophila melanogaster_. 


### References

1.	Cannan, W. J. and Pederson, D.S., “Mechanisms and Consequences of Double-Strand DNA Break Formation in Chromatin,”  Journal of Cellular Physiology, Vol. 231, No. 1, pp. 3-14, January 2016. doi: 10.1002/jcp.25048
2.	Hanscom, T. and McVey, M., “Regulation of Error-Prone DNA Double-Strand Break Repair and Its Impact on Genome Evolution,” Cells, Vol. 9, No. 7, July 2020. Doi: 10.3390/cells9071657

[My Final Report](files/finalreport.pdf)


## My Blog

I update [my weekly blog here](blog.html).
